aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -29,7 +29,13 @@ baseline audit of the accessibility of your application.
 
 The following collection of guides from the [HPE Product Accessibility Program Office](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Pages/index.aspx) help designers and developers create accessible products which meet the needs of HPE’s clients.
 
-<GuideCards />
+<GuideSection />
+
+## Accessible Product Design Videos
+
+A collection of internal short, educational videos from HPE explaining basic accessible design principles in software and web technologies. Each video is less than 10 minutes and is accompanied by a “Pocket Guide” with key takeaways that can be found under HPE [Accessible Design Training](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-design-training.aspx).
```

Comment:
This is a nice intro. Nice job setting it up and setting expectations for the reader. Just tweaking.

```suggestion
A collection of short educational videos from HPE explaining basic accessible design principles in software and web technologies.

Each video is less than 10 minutes and is accompanied by a “Pocket Guide” with key takeaways that can be found under HPE [Accessible Design Training](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-design-training.aspx).
```